### PR TITLE
Update project urls to match NPM url

### PR DESCRIPTION
**Pull request by sandersn**
Note that this *trivially* updates project urls by adding the NPM url to
the end, even when the urls are almost identical or the DT one is
outdated. I'll clean up the urls in a later commit.

Edit: This PR is ready to go in once I see that there aren't any weird errors from CI. I expect it will take too long to run for Travis, so I'm running it locally as well.

### CODE REVIEW

**Comment by sandersn** [comment]
Notes:

- atom is for the non-npm atom package, not the npm package atom, which is something to do with (a)synchronous programming. I'm not sure we have a good strategy for name collisions like this.

**Review by Igorbek — APPROVED** [review]

**Review by orta — APPROVED** [review]

**Comment by sandersn** [comment]
types/shipit should actually be shipit-cli or shipit-deploy; it prevents us from adding the real shipit npm module.

**Comment by sandersn** [comment]
types/titanium is actually for the non-npm package titanium-mobile. Both titanium and titanium-mobile are owned by appcelerator, though.

**Review by dolanmiu — APPROVED** [review]

**Comment by sandersn** [comment]
From my local run I got the following:

```
Error: The following packages had errors: d3-array, ember, jsdom, petit-dom, sinon, task-worklet
```

I'm pretty sure these are unrelated errors from typescript-next, but I'll confirm tomorrow before merging.

File: types/alexa-voice-service/index.d.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Type definitions for alexa-voice-service 0.0
-// Project: https://github.com/miguelmota/alexa-voice-service.js
```
Comment (dolanmiu) [inline types/alexa-voice-service/index.d.ts]: Why was this changed? 

https://github.com/miguelmota/alexa-voice-service.js is the actual URL

**Review by dolanmiu — CHANGES_REQUESTED** [review]

File: types/jsforce/index.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Type definitions for jsforce 1.9
-// Project: https://github.com/jsforce/jsforce
```
Comment (dolanmiu) [inline types/jsforce/index.d.ts]: Same for this.

Why change https to http?

**Review by dolanmiu — CHANGES_REQUESTED** [review]
Please make those changes

File: types/alexa-voice-service/index.d.ts (hunk 1 above)
Comment (LinusU) [reply types/alexa-voice-service/index.d.ts]: It's listed without `.js` in their package.json, seems like the repo was renamed since without it redirects to the new one. Submitted a PR here: https://github.com/miguelmota/alexa-voice-service.js/pull/15

File: types/jsforce/index.d.ts (hunk 2 above)
Comment (LinusU) [reply types/jsforce/index.d.ts]: https://github.com/jsforce/jsforce/pull/860

File: types/alexa-voice-service/index.d.ts (hunk 1 above)
Comment (sandersn) [reply types/alexa-voice-service/index.d.ts]: @dolanmiu when the urls are very similar and one redirects to the other, I kept the one from package.json because it's the source of truth, even when (1) not https or (2) missing a small name change. @LinusU has the right idea to fix it upstream.

**Comment by sandersn** [comment]
I filed Microsoft/TypeScript#29898 to track the failures -- although some were due to an old install of types-publisher on my machine.

**Comment by DanielRosenwasser** [comment]
OH MY GOD MAKE IT STOP

**Comment by orta** [comment]
Hah, you go @typescript-bot

**Comment by fenying** [comment]
800+ mails...
My god, R U crazy? @typescript-bot
